src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -85,6 +85,20 @@ public void addMessage(Message message) {
     return messagesInConversation;
   }
 
+  /** Find and return the Message with the given UUID. */
```

Comment:
Please add unit tests for these new methods

Reply:
The above method has been removed and will no longer require unit tests.

Reply:
Unit tests have been added.